src/main/java/com/paneedah/weaponlib/render/scopes/Reticle.java

```diff
@@ -11,11 +13,13 @@ public class Reticle {
 
     private static final int DEFAULT_TINT = 0x576574;
 
+    @Getter @Setter
     private ResourceLocation reticleTexture;
     private Type reticleType;
 
-
+    @Setter @Getter
     private float textureScale;
+    @Setter @Getter
     private Vec3d backgroundColor;
```

Comment:
Please set the annotations on the same line

Reply:
Done